src/deployment-guides/tko-aws.md

```diff
@@ -226,13 +226,15 @@ After doing the network configuration, complete the steps described in this sect
 
 4. Install Docker Engine on Ubuntu.  For installation instructions, see [Install Docker Engine on Ubuntu](https://docs.docker.com/engine/install/ubuntu/).
 
-5. Download the Tanzu CLI and other utilities for Linux from the Tanzu Kubernetes Grid [Download Product](https://customerconnect.vmware.com/downloads/details?downloadGroup=TKG-140&productId=988&rPId=73652) site.
+5. Download the Tanzu CLI and other utilities for Linux from the Tanzu Kubernetes Grid [Download Product](https://customerconnect.vmware.com/downloads/details?downloadGroup=TKG-142&productId=988&rPId=73652) site.
+
+	Note that the link above is for version 1.4.2 and the example commands below correspond to filenames and screens from that release. They may differ for other releases.
```

Comment:
@craigtracey I understand. The main issue addressed by the PR is that the screenshots and textual descriptions with them are outdated. I have not checked with TKO 1.4.0 but I assume it's unlikely that a patch release would have caused such a change (so they must have been wrong for 1.4.0 as well). However, having used the latest available 1.4.x release for my work, I thought it wise to update everything to that so that in case there might have been some differences in earlier releases.